apps/meteor/app/lib/server/methods/blockUser.ts

```diff
@@ -16,7 +16,7 @@ Meteor.methods({
 
 		const room = Rooms.findOne({ _id: rid });
 
-		if (!roomCoordinator.getRoomDirectives(room.t)?.allowMemberAction(room, RoomMemberActions.BLOCK)) {
+		if (!roomCoordinator.getRoomDirectives(room.t)?.allowMemberAction(room, RoomMemberActions.BLOCK, Meteor.userId() || undefined)) {
```

Comment:
Meteor.userId will never be null in this case, since it's checked first. You could assign its value to a constant and use only the constant in this method so that typescript will understand it is not nullable here.